mangum/adapter.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         self,
         app: ASGI,
         lifespan: LifespanMode = "auto",
-        api_gateway_base_path: str = "/",
```

Comment:
Perhaps we can still accept this with a warning for backwards compatibility ?

Reply:
Yeah, let’s emit a warning to switch to the new name to be removed in future release.